resources/js/features/events/components/AchievementSet/sortAchievements.test.ts

```diff
@@ -123,26 +123,30 @@ describe('Util: sortAchievements', () => {
     const eventAchievements: App.Platform.Data.EventAchievement[] = [
       {
         achievement: achievements[1],
-        activeFrom: new Date(now.getTime() - 1000).toISOString(),
-        activeThrough: new Date(now.getTime() + 1000).toISOString(),
+        activeFrom: new Date(now.getTime() - 1000).toISOString(), // 1 second ago
+        activeThrough: new Date(now.getTime() + 1000).toISOString(), // 1 second from now
+        activeUntil: new Date(now.getTime() + 1000).toISOString(), // 1 second from now - needed for activeUntil check
```

Comment:
Not sure if it affects the tests, but activeUntil will always be one day after activeThrough. In this case, you might want to just make it one second after.

Reply:
Still good to update in latest. Went ahead and also converted all these to use dayjs so they're more readable:
```ts
activeFrom: dayjs().subtract(1, 'second').toISOString(),
activeThrough: dayjs().add(1, 'second').toISOString(),
activeUntil: dayjs().add(1, 'second').add(1, 'day').toISOString(),
```